client/src/main/java/com/flipkart/flux/client/runtime/FluxRuntimeConnectorHttpImpl.java

```diff
@@ -149,6 +150,24 @@ public void submitScheduledEvent(String name, Object data, String correlationId,
         }
     }
 
+    @Override
+    public void submitEventUpdate(String name, Object data, String correlationId,String eventSource) {
```

Comment:
Can data be a generic class?

`public void <T> submitEventUpdate(String name, T data, String corelattionID, String eventSource) {`

Reply:
We could go generics, but currently everywhere Object notation is used, this refactoring can be added to Tech Debt and picked up in future.

Reply:
Can be done as part of refactoring exercise.